experiences/config/README.md

```diff
@@ -73,6 +73,16 @@ This directory contains the configuration files linked to each deployement. They
     * @type String
     */
   "filedrop": "https://example.com/filedrop",
+  /**
+    * Path or URL to override the default site logo
+    * @type String
+    */
+  "logo": "/some-other-logo.png",
+  /**
+    * Mapbox Token
+    * @type String
+    */
+  "mapboxToken": "pk.eyJ1IjoiYW5kcmVhc2t1bmRpZyIsImEiOiJja3ZxcnlmNXc2ZzUwMnFva2F2a3Q1azU5In0.NrvCU8OKlkwJOVFOgZzTzA",
```

Comment:
Is that our actual mapbox token? Maybe we shouldn't put that directly in the documentaiont?